chatbot/src/main/java/chatbot/core/classifier/Classifier.java

```diff
@@ -75,18 +96,20 @@ else if (queryIsPersonal(query)) {
 				log.info("HAWK!");
 				return new QAHandler();
 
+			} else if (queryisAutomated(query)) {				

```

Comment:
@Juzerhk Can you handle "Extract" related keyword logic from rivescript by using a new rivescript file for extraction? Because code is now modified by @divyasrinivasan1986 to be ML based and since its also a work in progress, you can work independently by modifying RivescriptOutputHandler and either of you don't need to merge repeatedly.